src/tintx/objects.py

```diff
@@ -10,6 +10,7 @@
 
 import numpy as np
 import pandas as pd
+from rasterio import features, transform  # type: ignore
```

Comment:
I think you don't need # type: ignore if you add 

```
[mpypy-resterio.*]
ignore_missing_imports = True
```

into the mypy.ini file.
```